aws-http-nio-client-netty/src/main/java/software/amazon/awssdk/http/nio/netty/NettySdkHttpClientFactory.java

```diff
@@ -78,6 +81,14 @@ private NettySdkHttpClientFactory(DefaultBuilder builder) {
     }
 
     /**
+     * @return The {@link EventLoopGroup} to use for the Netty client.
+     * @see Builder#eventLoopGroup(EventLoopGroup)
+     */
+    public Optional<EventLoopGroup> eventLoopGroup() {
```

Comment:
Feels weird that this is exposed as a `public` method on the factory - is the intent that it would be used to share the event loop group across multiple clients?